engine-src/Game/LambdaHack/Client/UI/HandleHelperM.hs

```diff
@@ -305,11 +301,12 @@ skillsOverlay aid = do
       skillValue = EM.singleton monoFont $ offsetOverlayX skValue
   return (EM.unionsWith (++) [skillLab, skillDescr, skillValue], kxs)
 
+-- extract a set of places for each level, counting number of each kind of PlaceEntry
```

Comment:
The comment I should have written is "Extract per-level statistics for each place kind, counting the number of occurrences of each type of @PlaceEntry@ for the given place kind on the given level".

Reply:
Wrong, it should be "Extract whole-dungeon statistics for each place kind, counting the number of occurrences of each type of @PlaceEntry@ for the given place kind and gathering the set of levels on which any entry for that place kind can be found".

Reply:
And the following context should precede the comment (and some of it should be a haddock of @PlaceEntry@ datatype, the first constructor of which should be renamed to @PEntrance@): "Places are rooms and other dungeon features, their names can be seen on a level map by aiming at a position that is an entry to the place (an individual entrance point, an approach area around the place or a phantom entry not on the map, but only used for statistics to witness the place exists), entries are proxies for initial places created on the level (which may be otherwise eradicated by burrowing the walls, etc.) and so used for dungeon statistics".

Reply:
And another piece of context: "The statistics are presented in the 'Dashboard/displace place lore' menu".